clang_delta/RemoveUnusedEnumMember.cpp

```diff
@@ -83,26 +89,30 @@ void RemoveUnusedEnumMember::HandleTranslationUnit(ASTContext &Ctx)
     TransError = TransMaxInstanceError;
     return;
   }
+  if (ToCounter != -1 && ToCounter > ValidInstanceNum) {
+    TransError = TransToCounterTooBigError;
+    return;
+  }
```

Comment:
Could we also check ```TransformationCounter > ValidInstanceNum``` and set ```TransError``` to ```TransMaxInstanceError``` it it's true? Thanks!

Reply:
It's already handled, see the code before this hunk.

Reply:
Ah, yes. I missed it since it wasn't shown in the diff. Sorry about that.